cmake/FindClangTidy.cmake

```diff
@@ -12,25 +12,27 @@
 # * ``ClangTidy_VERSION_MINOR`` The clang-tidy minor version
 # * ``ClangTidy_VERSION_PATCH`` The clang-tidy patch version
 
-find_program(ClangTidy_EXECUTABLE
-             NAMES clang-tidy
-                   clang-tidy-8
-                   clang-tidy-7
-                   clang-tidy-6.0
-                   clang-tidy-5.0
-                   clang-tidy-4.0
-                   clang-tidy-3.9
-                   clang-tidy-3.8
-                   clang-tidy-3.7
-                   clang-tidy-3.6
-                   clang-tidy-3.5
-             DOC "clang-tidy executable")
+find_program(
```

Comment:
You don't need this file since you have now hpc-coding-conventions, imo

Reply:
thank you! I created a new issue #109 for that!